googler

```diff
@@ -339,6 +403,17 @@ class GoogleParser(HTMLParser.HTMLParser):
 
         self.set_handlers_to('main')
 
+    ### Ignore list ###
+    IGNORE_LIST = [
+        # "People also ask"
+        # Sample query: VPN
+        # Screenshot: https://i.imgur.com/yfcsoQz.png
+        {
+            'tag': 'div',
```

Comment:
No, `div.related-question-part` is outside of the topmost result container `div.g`, so you have to watch for that in the `main` scope, which is the scope we're in for the majority of the time. Sure, I can add a conditional to check for it only in the `main` scope, but that means more complexity in the ignore list entry, and honestly, the ignore list is only useful for things that are in the main scope — we can avoid dropping into other scopes in the first place.

Also, as I mentioned, field test shows a consistent increase of <5ms in processing time, that means <0.1s even if we have 20 entries in the ignore list.